api/src/pcapi/core/offers/models.py

```diff
@@ -121,6 +121,7 @@ class Product(PcObject, Base, Model, HasThumbMixin, ProvidableMixin):
     isGcuCompatible: bool = sa.Column(sa.Boolean, default=True, server_default=sa.true(), nullable=False)
     isNational: bool = sa.Column(sa.Boolean, server_default=sa.false(), default=False, nullable=False)
     isSynchronizationCompatible: bool = sa.Column(sa.Boolean, default=True, server_default=sa.true(), nullable=False)
+    last_30_days_booking = sa.Column(sa.Integer, nullable=False, default=0)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Comme on en parlait, ça a plus de sens si c'est nullable
0 a un sens dans la colonne ; None = pas de valeur